src/cdk/table/sticky-styler.ts

```diff
@@ -242,8 +258,12 @@ export class StickyStyler {
    * to be sticky (and -webkit-sticky), setting the appropriate zIndex, and adding a sticky
    * direction and value.
    */
-  _addStickyStyle(element: HTMLElement, dir: StickyDirection, dirValue: number) {
+  _addStickyStyle(element: HTMLElement, dir: StickyDirection, dirValue: number,
```

Comment:
I'm not a huge fan of boolean, "do something extra" flags like this, but I can live with it since it's an internal  method. I think ideally we'd have a data structure to pass to this method that had these args.

Reply:
From a readability perspective I agree, though from a function that gets called hundreds or thousands of times while setting up a table perspective, I'm not sure it's the right trade-off.